samcli/lib/build/workflow_config.py

```diff
@@ -278,6 +207,11 @@ def get_workflow_config(
 
         # Identify workflow configuration from the workflow selector.
         config = cast(WorkFlowSelector, selector).get_config(code_dir, project_dir)
+
+        EventTracker.track_event("WorkflowUsed", f"{config.language}-{config.dependency_manager}")
+        if config.dependency_manager == "npm-esbuild":
```

Comment:
Please confirm with Wing if we need this. Otherwise let's get rid of this `UsedFeature` metric since we can get this same data from the `WorkflowUsed` metric.

Reply:
👍